Readme.md

```diff
@@ -38,6 +38,11 @@ app.use(logger())
   Recommended that you `.use()` this middleware near the top
   to "wrap" all subsequent middleware.
 
+## Options
+
+* `logger` - an object implementing, at least, a `log` function with variadic arguments. (default `console`)
```

Comment:
I'd vote for having a middleware that emits an object with `method, url, status, duration, size` etc, maybe we build this one on top of that one. Using the arbitrary strings we have in this module for reporting would be to restrictive to commit to it api-wise personally (even nitpicky stuff like "log" not really being a severity level)